presto-docs/src/main/sphinx/admin/verifier.rst

```diff
@@ -290,6 +292,16 @@ Name                                        Description
                                             It only applies to ``Insert`` and ``CreateTableAsSelect`` queries.
                                             It would verify each partition's data checksum if the inserted table is partitioned.
                                             It would verify each bucket's data checksum if the inserted table is bucketed.
+``function-substitutes``                    Specification of function substitutions, in the format of
+                                            ``/foo(c0,_)/bar(c0)/,/fred(c0,c1)/baz(qux(c1,c0))/,/foobar(c0)/if(qux(c1),bar(c0),baz(c1))/,...``,
+                                            where ``foo(c0, _)`` would be substituted by ``bar(c0)``, with the declared arguments
+                                            applied to the corresponding positions. Concatenate function substitutions with comma.
+
+                                            Make sure to select a function substitute that has the return type and argument types
+                                            compatible with those of the original function, to produce a valid source query. For
+                                            example, ``/array_agg(z)/array_sort(array_agg(z))/,/approx_percentile(x,y)/avg(x)/``
+                                            Declare the function arguments as identifiers if they need to be applied to
```

Comment:
```suggestion
                                            
                                            Declare the function arguments as identifiers if they need to be applied to
```
Same line break. 

Here's a screenshot of my local build with these suggestions. 
<img width="796" alt="Screenshot 2024-05-20 at 11 03 31 AM" src="https://github.com/prestodb/presto/assets/7013443/f7f96cdd-b31e-4b9f-b1bc-a42684de70e1">